MapboxCoreNavigation.podspec

```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
   s.name = "MapboxCoreNavigation"
-  s.version = "0.29.1"
+  s.version = "0.30.0"
```

Comment:
Fixed the version number in the podspecs in efce37be1ef468699c66a04e1a528deea384784c, since CocoaPods is looking for specific tags based on this version string.